### PR TITLE
Adds brass, used to construct various brass objects

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscadd: Added brass, producible by proselytizing rods, metal, or plasteel, or by using Replicant Alloy in-hand, and usable to construct a variety of cult-y brass objects.
tweak: Objects constructable with brass include; wall gears, pinion airlocks, brass windoors, brass windows, brass table frames and tables, and brass floor tiles.
rscadd: Cogscarabs can convert brass sheets into liquified alloy with their proselytizer.
tweak: You can use brass sheets on wall gears to produce clockwork walls, or, if the gear is unanchored, false clockwork walls.
fix: Fixed a bug preventing you from converting grilles into ratvar grilles.
experiment: Please note that conservation of mass exists.
/:cl:

### CODE REVIEW

**Comment by nullbear** [comment]
OH. MY. GOD. 

UGH.

WHERE DO YOU GET THE BRASS THO. SAME QUESTION FOR ALUMINUM + COPPER

**Comment by evsey9** [comment]
tl;dr non-antag clock cult stuff?

**Comment by nullbear** [comment]
no evsey9 can only be made by clockcult

**Comment by ChangelingRain** [comment]
Non-servants CAN use the brass but it's not really very useful because it's just slightly stronger windows/tables and fancy walls, and if there's a clockcult it helps them because oh look you're bait for sec

**Comment by nullbear** [comment]
And since the only way to get it is during clock cult -> Using it as a non cultist means you're helping them.

File: code/game/objects/structures/false_walls.dm [hunk 1]
```diff
@@ -272,51 +257,62 @@
...
 
 /obj/structure/falsewall/abductor
 	name = "alien wall"
 	desc = "A wall with alien alloy plating."
 	icon = 'icons/turf/walls/abductor_wall.dmi'
 	icon_state = "abductor"
-	mineral = "abductor"
-	walltype = "abductor"
+	mineral = /obj/item/stack/sheet/mineral/abductor
+	walltype = /turf/closed/wall/mineral/abductor
```
Comment (xxalpha) [inline code/game/objects/structures/false_walls.dm]: These walltype vars are not used for anything, stop perpetuating this awful meme.

File: code/game/objects/structures/false_walls.dm [hunk 2]
```diff
@@ -87,10 +91,7 @@
 
 /obj/structure/falsewall/proc/ChangeToWall(delete = 1)
 	var/turf/T = get_turf(src)
-	if(!walltype || walltype == "metal")
-		T.ChangeTurf(/turf/closed/wall)
-	else
-		T.ChangeTurf(text2path("/turf/closed/wall/mineral/[walltype]"))
+	T.ChangeTurf(walltype)
```
Comment (ChangelingRain) [inline code/game/objects/structures/false_walls.dm]: I DON'T BELIEVE YOU I DON'T BELIEVE YOU

Comment (xxalpha) [inline code/game/objects/structures/false_walls.dm]: I think I am confusing this var with another. :v:

Comment (ChangelingRain) [inline code/game/objects/structures/false_walls.dm]: seriously it being this stupid texttopath stuff was awful and super difficult to work with/around YOU HAVE THE VAR JUST MAKE THE VAR A TYPE YOU SHITLORD

Comment (xxalpha) [inline code/game/objects/structures/false_walls.dm]: Ah yes, walls have this fucking var too for some reason. The same name.

Comment (ChangelingRain) [inline code/game/objects/structures/false_walls.dm]: THERE IS NO GOOD REASON TO DO THIS DYNAMIC BULLSHIT WHEN YOU HAVE A VAR THAT CAN SERVE AS A TYPE INSTEAD LIKE WHO THE FUCK CODED THIS

Comment (xxalpha) [inline code/game/objects/structures/false_walls.dm]: Remove the wall var because it was probably being used to pass their type onto falsewalls.

File: code/game/objects/structures/false_walls.dm [hunk 3]
```diff
@@ -117,32 +118,28 @@
...
 	else if(istype(W, /obj/item/weapon/pickaxe/drill/jackhammer))
 		var/obj/item/weapon/pickaxe/drill/jackhammer/D = W
 		D.playDigSound()
-		dismantle(user)
+		dismantle(user, TRUE)
 	else
 		return ..()
 
-/obj/structure/falsewall/proc/dismantle(mob/user)
+/obj/structure/falsewall/proc/dismantle(mob/user, girder)
```
Comment (phil235) [inline code/game/objects/structures/false_walls.dm]: I'd prefer a better arg name, "girder" makes me think the arg is supposed to be a girder object. "drop_girder" is more explicit.

File: code/game/objects/structures/table_frames.dm [hunk 4]
```diff
@@ -125,7 +125,20 @@
...
+	if(istype(I, /obj/item/stack/sheet/brass))
+		var/obj/item/stack/sheet/brass/W = I
+		if(W.get_amount() < 1)
+			user << "<span class='warning'>You need one brass sheet to do this!</span>"
+			return
+		user << "<span class='notice'>You start adding [W] to [src]...</span>"
+		if(do_after(user, 20, target = src) && W.use(1))
+			new /obj/structure/table/reinforced/brass(src.loc)
+			qdel(src)
+	return 1
```
Comment (phil235) [inline code/game/objects/structures/table_frames.dm]:
```
else
	return ..()
```

**Comment by phil235** [comment]
Except for that one comment, the code is good to go.
